balldetection.py

```diff
@@ -0,0 +1,64 @@
+import cv2
+import numpy as np
+import math
+
+koordinatni_pocetak = (0, 0)
+
+def click(event, x, y, flags, param):
+    global koordinatni_pocetak
+    if event == cv2.EVENT_LBUTTONDOWN:
+        koordinatni_pocetak = (x, y)
+
+def get_polozaj():
+    global koordinatni_pocetak, roi_x, roi_y, roi_h, roi_w, frame
```

Comment:
Nemojte koristiti global. Kada budete ubacili klase ovo vam neće trebati.
Samo stavite da `get_polozaj(frame)` vraća `angle`. A sve ostalo što vam treba od promenljivih nek bude polje unutar klase.